packs/caddy/variables.hcl

```diff
@@ -0,0 +1,93 @@
+variable "job_name" {
+  description = "The name to use as the job name which overrides using the pack name"
+  type        = string
+  // If "", the pack name will be used
+  default = ""
+}
+
+variable "datacenters" {
+  description = "A list of datacenters in the region which are eligible for task placement"
+  type        = list(string)
+  default     = ["dc1"]
+}
+
+variable "region" {
+  description = "The region where the job should be placed"
+  type        = string
+  default     = "global"
+}
+
+variable "namespace" {
+  description = "The namespace where the job should be placed."
+  type        = string
+  default     = "default"
+}
+
+variable "consul_service_name" {
+  description = "The consul service you wish to load balance"
+  type        = string
+  default     = "webapp"
+}
+
+variable "version_tag" {
+  description = "The docker image version. For options, see https://hub.docker.com/_/caddy"
+  type        = string
+  default     = "2.4.5"
+}
+
+variable "http_port" {
+  description = "The Nomad client port that routes HTTP traffic to Caddy."
+  type        = number
+  default     = 8443
+}
+
+variable "https_port" {
+  description = "The Nomad client port that routes HTTPS traffic to Caddy."
+  type        = number
+  default     = 8080
+}
```

Comment:
It might just be me here, but I find it confusing http uses 8443 and https 8080 rather than the other way around.